Critical bug: Consumer didn't build correctly/won't start

The GitHub action has built the new consumer, but it will not start up.

```
consumer-1  | node:internal/modules/cjs/loader:1147
consumer-1  |   throw err;
consumer-1  |   ^
consumer-1  | 
consumer-1  | Error: Cannot find module '/app/dist/index.cjs'
consumer-1  |     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
consumer-1  |     at Module._load (node:internal/modules/cjs/loader:985:27)
consumer-1  |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
consumer-1  |     at node:internal/main/run_main_module:28:49 {
consumer-1  |   code: 'MODULE_NOT_FOUND',
consumer-1  |   requireStack: []
consumer-1  | }
consumer-1  | 
consumer-1  | Node.js v20.11.1
consumer-1 exited with code 0
```

Using current code base:

```
  consumer:
    image: gabisonfire/knightcrawler-consumer:latest
    env_file:
      - .env
    deploy:
      replicas: ${CONSUMER_REPLICAS}
    <<: *knightcrawler-app
    networks:
      - knightcrawler-network
      - postgres-access
```

--- a/src/node/consumer/Dockerfile
+++ b/src/node/consumer/Dockerfile
@@ -26,6 +26,6 @@ RUN chmod u-s /usr/bin/wall /usr/bin/expiry /usr/bin/chage /usr/bin/passwd /usr/
 USER consumer
 
 EXPOSE 7001
-ENTRYPOINT [ "node", "dist/index.cjs" ]
+ENTRYPOINT [ "node", "dist/main.cjs" ]
 HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 \
     CMD pgrep -x node > /dev/null || exit 1

